geo/src/algorithm/is_convex.rs

```diff
@@ -245,4 +262,35 @@ mod tests {
         assert!(!two.is_strictly_ccw_convex());
         assert!(!two.is_strictly_cw_convex());
     }
+
+    #[test]
+    fn test_duplicate_pt() {
+        let ls_unclosed: LineString<f64> = wkt! (LINESTRING (0 0, 1 1, 2 0)).convert();
```

Comment:
Nice use of `convert`! I've always added all the decimal points to every coord, but it's annoying. I'm going to steal this for sure.